<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSection</class>
 <widget class="QWidget" name="WidgetSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;
color:white;
font-family:&quot;맑은 고딕&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDoubleSpinBox{
color:white;
border:1px solid #494949;
background-color:black;
}

QDoubleSpinBox::up-button {
	width:30px;
}

QDoubleSpinBox::down-button {
	width:30px
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	font-family:&quot;맑은 고딕&quot;;
	background-color:#494949;
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 190, 11, 255), stop:1 rgba(251, 86, 7, 255));
	border:none;
}</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDoubleSpinBox{
color:white;
border:1px solid #494949;
background-color:black;
}

QDoubleSpinBox::up-button {
	width:30px;
}

QDoubleSpinBox::down-button {
	width:30px
}</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:&quot;맑은 고딕&quot;;</string>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDoubleSpinBox{
color:white;
border:1px solid #494949;
background-color:black;
}

QDoubleSpinBox::up-button {
	width:30px;
}

QDoubleSpinBox::down-button {
	width:30px
}</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:&quot;맑은 고딕&quot;;</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:&quot;맑은 고딕&quot;;</string>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	font-family:&quot;맑은 고딕&quot;;
	background-color:#494949;
}

QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 190, 11, 255), stop:1 rgba(251, 86, 7, 255));
	border:none;
}</string>
     </property>
     <property name="text">
      <string>Voxel Section</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
